src/ast.rs

```diff
@@ -256,5 +256,6 @@ pub enum Statement<'a> {
 pub struct Chunk<'a> {
     #[serde(borrow)]
     pub statements: Vec<Statement<'a>>,
+    #[serde(rename = "return")]
     pub r#return: Option<Vec<Expression<'a>>>,
```

Comment:
We should just give this a different name instead of raw-quoting the key.

Reply:
return_value?